genome_designer/genome_finish/insertion_placement.py

```diff
@@ -0,0 +1,201 @@
+from Bio import SeqIO
```

Comment:
third-party imports go in second cluster of imports (first cluster is native python libraries, second cluster is third-party library imports, third cluster is imports from the present project)